ledger/participant-integration-api/src/test/lib/scala/platform/store/dao/JdbcLedgerDaoBackend.scala

```diff
@@ -43,7 +43,7 @@ private[dao] trait JdbcLedgerDaoBackend extends AkkaBeforeAndAfterAll { this: Su
   override protected def beforeAll(): Unit = {
     super.beforeAll()
     implicit val executionContext: ExecutionContext = system.dispatcher
-    resource = newLoggingContext { implicit logCtx =>
+    resource = newLoggingContext { implicit loggingContext =>
```

Comment:
See if it could make sense to use `LoggingContext.ForTesting` to remove a level of nesting.

Reply:
Nah. :slightly_smiling_face: